gpytorch/lazy/lazy_tensor.py

```diff
@@ -1531,6 +1556,36 @@ def sum(self, dim=None):
         else:
             raise ValueError("Invalid dim ({}) for LazyTensor of size {}".format(orig_dim, self.shape))
 
+    def svd(self):
+        """
+        Compute the SVD of the lazy tensor `M` s.t. `M = U @ S @ V.T`.
+        This can be very slow for large tensors. Should be special-cased for tensors with particular structure.
+
+        Returns:
+            :obj:`torch.Tensor`:
+                The left singular vectors (`U`).
+            :obj:`torch.Tensor`:
+                The singular values (`S`).
+            :obj:`torch.Tensor`:
+                The right singular vectors (`V`).
+        """
+        return torch.svd(self.evaluate())
```

Comment:
We'll want to return `U` and `V` here as `LazyTensor` objects, as for Kronecker tensors these can be themselves expressed as `KroneckerProductLazyTensor`s. Same for the eigenvectors in `symeig`.